app/assets/stylesheets/application.css

```diff
@@ -1,17 +1,20 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- *= require foundation_and_overrides
+body{
```

Comment:
```
It is generally better to create a new file per style scope.
```

Styles don't belong in this file! This manifest file, while it looks like is all code comments, is actually serving a purpose. In the `stylesheets` folder we should create separate files for our custom CSS, like creating a `main.scss` (I recommend SCSS over CSS for "super powers"). No imports or require statements are necessary when creating new CSS files - the purpose of this manifest file is to require the files for us.

The code comments that were removed need to be put back - particularly because of the `require` statements. One change you'll want to make is in the ordering of the require statements (the order gets messed up from make_it_so):
```
 *= require foundation_and_overrides
 *= require_tree .
 *= require_self
```
when this file is evaluated it requires all Foundation styles, then all of your custom CSS files in the `tree` (aka stylesheets folder), then lastly it requires this file. No styles should be added to this file, it's intended to be an absolute last resort, and even then we should consider separate stylesheets so that our CSS is organized.